lib/tasks/ita.rake

```diff
@@ -23,19 +23,12 @@ namespace :ita do
     ImportEmptyIndices.call
   end
 
-  def importers(module_or_model_class)
-    if module_or_model_class.is_a?(Indexable)
-      model_class = module_or_model_class
-      [model_class.importer_class]
+  def importers(module_or_importer_class_name)
+    module_or_importer_class = module_or_importer_class_name.constantize
+    if module_or_importer_class.include?(Importer)
+      [module_or_importer_class]
     else
-      modu1e = module_or_model_class
-      module_importer_files =
-        "#{Rails.root}/app/importers/#{modu1e.name.typeize}/*"
-      Dir[module_importer_files].each { |f| require f }
-
-      modu1e.constants
-        .map { |constant| modu1e.const_get(constant) }
-        .select { |klass| klass.include?(Importer) }
+      module_or_importer_class.importers
```

Comment:
@tmhammer you don't need to extend all importer classes with the `Importers` module; the above will do the job.